book/src/comparisons.md

```diff
@@ -0,0 +1,77 @@
+# Comparisons
+
+Within the Cardano community there has been a florishing ecosystem
+of alternative langauges for writing smart contracts. So naturally, one might ask about
+the differences between these and which they should use for their use case. There is
+also a big misconception about how writing smart contracts actually works on Cardano. In this document,
+we'll list some of the main alternatives along with their differences and similarities. Before we get into this though, let's discuss the misconception first so everyone is on the same page.
+
+## The Misconception
+
+**Cardano uses Haskell for smart contracts**
+
+This is absolutely **FALSE**.
```

Comment:
I don't agree with this. PlutusTx is almost Haskell proper.

Reply:
Agreed